docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -37,37 +38,39 @@ Custom Search Attributes of the `Text` type cannot be used in **ORDER BY** claus
 
 ### Partial string match
 
-The `=` operator works like **CONTAINS** to find Workflows with Search Attributes that contain a specific word.
+The `=` operator functions similarly to **CONTAINS**, helping to find Workflows with Search Attributes containing a specific word.
+Partial string matching only applies to locating Text Search Attributes.
 
 <!-- note: advanced vis features will be supported in SQL upon the release of v1.20.-->
 
-For example, if you have a custom Search Attribute named `Description` of `Text` type with the value of "The quick brown fox jumps over the lazy dog", searching for `Description='quick'` or `Description='fox'` will successfully return the Workflow.
-However, partial word searches such as `Description='qui'` or `Description='laz'` will not return the Workflow.
-This is because [Elasticsearch's tokenizer](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-standard-tokenizer.html) is configured to return complete words as tokens.
+For instance, if you have a custom Search Attribute named `Description` of `Text` type with the value "The quick brown fox jumps over the lazy dog", a search for `Description='quick'` or `Description='fox'` will successfully return the Workflow.
+However, searches for partial words like `Description='qui'` or `Description='laz'` won't return the Workflow.
+This limitation arises because [Elasticsearch's tokenizer](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-standard-tokenizer.html) is configured to return complete words as tokens.
 
 ### Efficient API usage
 
-An Advanced List Filter API may take longer to respond if it is retrieving a large number of Workflow Executions (over 10,000).
+If the Advanced List Filter API retrieves a substantial number of Workflow Executions (over 10,000), the response time might be longer.
 
-With Temporal Server v1.20 and later, you can use the `CountWorkflow` API to efficiently count the number of [Workflow Executions](/concepts/what-is-a-workflow-execution).
+Starting from Temporal Server v1.20 and later, you can employ the `CountWorkflow` API to efficiently count the number of [Workflow Executions](/concepts/what-is-a-workflow-execution).
 
-Paginate the results with the `ListWorkflow` API by using the page token to retrieve the next page; continue until the page token is `null`/`nil`.
+To paginate the results using the `ListWorkflow` API, use the page token to retrieve the next page.
+Continue until the page token becomes `null`/`nil`.
 
 #### List Filter examples
 
-The following is a List Filter set with [`tctl`](/tctl-v1/workflow/list):
+Here are examples of List Filters set with [`tctl`](/tctl-v1/workflow/list):
```

Comment:
If you're referring to this entire section, change the ending colon to a period to avoid implying that the following code block contains multiple examples.

Reply:
note to self: remember to change this once `tctl` is officially deprecated